### PR TITLE
LSE-373: workqueue: Fix errors seen for newer (v6.4 and later) kernels.

### PR DESCRIPTION
**Pull request by imran-kn**
LSE-373.

### CODE REVIEW

File: drgn_tools/workqueue.py [hunk 1]
```diff
@@ -63,6 +94,33 @@ def _print_work(work: Object) -> None:
...
+    # we can track both name change and type change of this member.
+    try:
+        pwq = per_cpu_ptr(workqueue.cpu_pwqs, cpu)
+    except AttributeError:
+        prog = workqueue.prog_
+        pwq = Object(
+            prog,
+            "struct pool_workqueue",
+            address=prog.read_word(per_cpu_ptr(workqueue.cpu_pwq, cpu)),
+        ).address_of_()
```
Comment (brenns10) [inline drgn_tools/workqueue.py:119]: You can simplify this by just dereferencing the double pointer.
```suggestion
        pwq = per_cpu_ptr(workqueue.cpu_pwq, cpu)[0]
```

**Review by brenns10 — CHANGES_REQUESTED** [review]
I tested this on UEK-next and it resolves all the workqueue related issues. Looks to pass on gitlab CI too. However, one tiny nit if you don't mind. Thanks!

File: drgn_tools/workqueue.py (hunk 1 above)
Comment (imran-kn) [reply drgn_tools/workqueue.py]: Makes sense. Have done that now.

**Review by brenns10 — APPROVED** [review]
Thanks for the change, everything else is good! The test failure is unrelated to your change.

**Pull request merged**